qa-covid19.planx-pla.net/portal/gitops.json

```diff
@@ -157,8 +158,7 @@
         "dataType": "dataset",
         "nodeCountTitle": "Datasets",
         "fieldMapping": [
-          { "field": "name", "name": "Dataset" },
```

Comment:
Should keep this field mapping but change to Dataset Name for consistency?

Reply:
in "Datasets" tab i am naming fields "Name, Source" and in other tabs i am naming them "Dataset Name, Dataset Source"

Reply:
Thanks for the explanation

Reply:
thanks for the thorough review :D